Tests/AlgoliaSearch/ReindexCommandTest.php

```diff
@@ -83,7 +86,7 @@ public function reIndex($use_entity_alias, $safe = true)
 
         $input = new ArrayInput($options);
 
-        $this->command->run($input, new ConsoleOutput());
+        $this->command->run($input, new NullOutput());
```

Comment:
Reverted my changes to `ConsoleOutput` as suggested by you. Will try to investigate your errors in a new pull request. Are your local errors the same as the one's on Travis? (`No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.`) Just to be sure to search for the same thingies 😄.